### PR TITLE
Api console refactor

### PR DESCRIPTION
**Pull request by osisoft-mbishop**
@matthewcharles27 , I refactored the API Console from a task-based topic to a concept/reference-based topic, which is more representative of the user experience. Developers will already be familiar with REST APIs and will make small incremental changes.
The markdown diffs make it a little difficult to read, so you might want to review on the dev site: https://aveva-dev.zoominsoftware.io/bundle/data-hub-api-console-refactor/page/api-reference/api-console.html

Also, here's the currently published version of the topic for comparison: https://docs.aveva.com/bundle/data-hub/page/api-reference/api-console.html

### CODE REVIEW

File: content/api-reference/api-console.md [hunk 1]
```diff
@@ -4,61 +4,115 @@ uid: apiConsole
...
 # API console
 
-The API Console provides a graphical interface for using the REST API. When using the API Console, you select a GET, POST, PUT, DELETE, or PATCH action, select the objects for the action, and execute.
+The API Console provides a graphical interface for using the AVEVA Data Hub REST API. Developers can use this console to configure and test API requests before implementing them in their own applications. To use the API Console, select select **Developer Tools** > **API Console** from the left pane.
 
-To use the API Console:
+## Request area
 
-1. In the left pane, select **Developer Tools** > **API Console**.
+Use the request area to configure an API request. Use this space to choose a version of the API, the request scope, the request verb, and the URI endpoint. This area is also used to configure the request header, body, and parameters.
```
Comment (matthewcharles27) [inline content/api-reference/api-console.md:11]:
```suggestion
Use the request area to configure an API request. Choose a version of the API, the request scope, the request verb, and the URI endpoint. This area is also used to configure the request header, body, and parameters.
```

File: content/api-reference/api-console.md [hunk 2]
```diff
@@ -4,61 +4,115 @@ uid: apiConsole
...
+### API version
 
-1. Select an action from the **GET/POST/PUT/DELETE/PATCH** dropdown list.
+Select a version from the dropdown list. For more information on the available versions, see [AVEVA Data Hub API versioning](xref:osisoftCloudServices#aveva-data-hub-api-versioning).
 
-1. Type a slash (/) in the text field to display a dropdown list.
+### Tenant path
 
-1. If you selected a tenant-scoped path in the second step, you can take any of the following actions:
+Select the **Root/Tenant path enabled** ![root/tenant path enabled](../_icons/default/home.svg) icon to toggle between a tenant-scoped path or a root-scoped path. The **Full Path** field (which is readonly) updates according to your selection.
```
Comment (matthewcharles27) [inline content/api-reference/api-console.md:21]: Is the difference between tenant-scope and root-scoped considered common knowledge or should you define that?

File: content/api-reference/api-console.md [hunk 3]
```diff
@@ -4,61 +4,115 @@ uid: apiConsole
...
+- `POST`: Adds new data.
+- `PUT`: Replaces existing data.
+- `DELETE`: Deletes existing data.
+- `PATCH`: Updates existing data.
 
-  The **Full Path** field shows an encrypted version of the selections you have made. Based on your selections, other fields may also be automatically prepended to the path. Together, these fields produce a path to a REST endpoint.
+### URI
 
-  **Tip:** Namespace- and community-scoped API requests autosuggest resource identifiers. For more information, see [Path autocomplete](#path-autocomplete).
+Use the **URI** field to enter an API route to make a request against. If your request is tenant-scoped, you can type a forward slash (/) in the field to display a dropdown list of path autocomplete options. If you selected a root-scoped path, a different dropdown list gives you other categories of data.
```
Comment (matthewcharles27) [inline content/api-reference/api-console.md:37]:
```suggestion
Use the **URI** field to enter an API route to make a request against. If your request is tenant-scoped, you can type a forward slash (/) in the field to display a dropdown list of path autocomplete options. If you select a root-scoped path, a different dropdown list gives you other categories of data.
```

File: content/api-reference/api-console.md [hunk 4]
```diff
@@ -4,61 +4,115 @@ uid: apiConsole
...
 
-1. Select the blue button in the lower right, which shows the action you selected earlier.
+For a complete reference of URI endpoints available in the AVEVA Data Hub API, refer to the [API Reference](xref:osisoftCloudServices).
 
-  The request to the endpoint is issued based on the path you have constructed. For **POST** and **PUT** requests, the **Body** entry field displays. Type the request body into the **Body** entry field.
+#### Path autocomplete
 
-  **History** records the calls that you make so you can replay them again.
+When entering a URI for an API request, the API console automatically suggests available identifiers for path completion. Additionally, a chip next to the identifier displays the resource name that it resolves to. Resources are listed in alphabetical order.
```
Comment (matthewcharles27) [inline content/api-reference/api-console.md:43]:
```suggestion
When entering a URI for an API request, the API console automatically suggests available identifiers for path completion. Additionally, a chip next to the identifier displays the resource name to which it resolves. Resources are listed in alphabetical order.
```

**Review by matthewcharles27 — COMMENTED** [review]
Just a few little things. Looks good to me.

File: content/api-reference/api-console.md (hunk 2 above)
Comment (osisoft-mbishop) [reply content/api-reference/api-console.md:21]: I'm going to need to ask around a bit to find out the difference.

**Pull request merged**